src/pynwb/ophys.py

```diff
@@ -144,7 +144,7 @@ class ROI(NWBContainer):
             {'name': 'pix_mask', 'type': Iterable, 'doc': 'List of pixels (x,y) that compose the mask.'},
             {'name': 'pix_mask_weight', 'type': Iterable, 'doc': 'Weight of each pixel listed in pix_mask.'},
             {'name': 'img_mask', 'type': Iterable, 'doc': 'ROI mask, represented in 2D ([y][x]) intensity image.'},
-            {'name': 'reference_images', 'type': ImageSeries, 'doc': 'One or more image stacks that the masks apply to (can be oneelement stack).'})
+            {'name': 'reference_images', 'type': (ImageSeries, str), 'doc': 'One or more image stacks that the masks apply to (can be oneelement stack).'})
```

Comment:
@nicain Just to confirm, the implementation to support both image series and image is available ?